kubernetes/helm/collabora-online/values.yaml

```diff
@@ -61,17 +62,17 @@ environmentVariablesLool:
   - name: extra_params
     value: --o:ssl.enable=false
   - name: username
-    value: 
+    value: peter
```

Comment:
default user/password? I'd not use them.

Reply:
i will remove them